.github/workflows/pr-lint.yml

```diff
@@ -13,11 +13,36 @@ permissions:
 jobs:
   main:
     permissions:
-      pull-requests: read  # for amannn/action-semantic-pull-request to analyze PRs
+      # for amannn/action-semantic-pull-request to analyze PR titles
+      # for marocchino/sticky-pull-request-comment to add comments to the PR
+      pull-requests: write
       statuses: write  # for amannn/action-semantic-pull-request to mark status of analyzed PR
     name: Validate PR title
     runs-on: ubuntu-latest
     steps:
       - uses: amannn/action-semantic-pull-request@v5
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: marocchino/sticky-pull-request-comment@v2
+        # When the previous steps fails, the workflow would stop. By adding this
+        # condition you can continue the execution with the populated error message.
```

Comment:
Does this comment make sense?

Reply:
It's just saying that this step runs even if the previous step fails.